### PR TITLE
Add a --quiet option to remove all logs

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Review by iomaganaris — APPROVED** [review]
LGTM
Could you just please add this option in the root `README.md` to keep all the options in place?

File: src/nmodl/main.cpp [hunk 1]
```diff
@@ -158,7 +161,10 @@ int main(int argc, const char* argv[]) {
     app.get_formatter()->column_width(40);
     app.set_help_all_flag("-H,--help-all", "Print this help message including all sub-commands");
 
-    app.add_flag("-v,--verbose", verbose, "Verbose logger output")->ignore_case();
+    auto v_flag = app.add_flag("-v,--verbose", verbose, "Verbose logger output")->ignore_case();
+    auto q_flag = app.add_flag("-q,--quiet", quiet, "Remove logger output")->ignore_case();
+    v_flag->excludes(q_flag);
```
Comment (pramodk) [inline src/nmodl/main.cpp:166]: @alkino : is there a way to use existing verbose flag for the same purpose? (i.e. having different levels of verbose enabled?)

Comment (alkino) [reply src/nmodl/main.cpp:166]: --quiet --verbose=2 --debug?
Those 3 flags?

**Comment by alkino** [comment]
maybe --quiet should always print warning and above ?

File: src/nmodl/main.cpp (hunk 1 above)
Comment (pramodk) [reply src/nmodl/main.cpp:166]: no, just verbose with different levels and we can get rid of --debug and --quiet.

**Comment by pramodk** [comment]
> maybe --quiet should always print warning and above ?

Make sense to me.

**Review by pramodk — APPROVED** [review]
LGTM, thanks!

**Pull request merged**